lib/pluggy_elixir/http_adapter/response.ex

```diff
@@ -0,0 +1,17 @@
+defmodule PluggyElixir.HttpAdapter.Response do
+  @moduledoc """
+  This module defines a `PluggyElixir.HttpAdapter.Response.t/0` struct that stores http responses.
+  """
+
+  defstruct [:body, :headers, :status]
```

Comment:
Same as [here](https://github.com/brainnco/pluggy_elixir/pull/1/files#r620319706)

Missing `@enforce_keys`, what do you think about it?